client/components/deposits-overview/index.tsx

```diff
@@ -49,9 +49,10 @@ const DepositsOverview: React.FC = () => {
 
 	const availableFunds = overview?.available?.amount ?? 0;
 	const pendingFunds = overview?.pending?.amount ?? 0;
+	const totalFunds = availableFunds + pendingFunds;
```

Comment:
Interesting to see this `Total balance` concept emerging in the code!

- #8070